.circleci/config.yml

```
@@ -543,6 +543,21 @@ jobs:
       - checkout
       - setup_environment:
           platform: << parameters.platform >>
+      - when:
+          condition:
+            not: << pipeline.parameters.nightly >>
+          steps:
+            - run:
+                command: >
+                  # Validate that version in the Cargo.toml matches the tag version
+                  # Fail the release if the versions don't match
+                  # Note that the tag has the prefix `v`, so this needs to be used to prefix the VERSION variable.
+                  VERSION=v$(cargo metadata --format-version=1 --no-deps | jq --raw-output '.packages[0].version')
```

This should be quoted, per best practices, since it could end up with unknown input.

```suggestion
                  VERSION="v$(cargo metadata --format-version=1 --no-deps | jq --raw-output '.packages[0].version')"
```

---

Are we sure that `jq` exists here?